src/components/MemberComponent.js

```diff
@@ -0,0 +1,317 @@
+import React, { useReducer, useEffect } from "react";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  List,
+  Modal,
+  Typography,
+  Button,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
+  IconButton,
+  Container,
+  TextField,
+} from "@material-ui/core";
+import DeleteIcon from "@material-ui/icons/Delete";
+import AddIcon from "@material-ui/icons/Add";
+import * as db from "../dbUtils";
+
+const MemberComponent = (refreshProjects, project, displayPopup) => {
```

Comment:
I seem to be getting warnings on compile with these props. not sure how to take care of them

Reply:
if it works, warnings aren't really a problem. If they seem dire can you copy paste them here?